Tests/Test_SendTritium.c

```diff
@@ -0,0 +1,337 @@
+#include "Minions.h"
+#include "Pedals.h"
+#include "FaultState.h"
+#include "CANbus.h"
+#include "MotorController.h"
+#include "ReadCarCAN.h"
+#include "BSP_UART.h"
+#include "Tasks.h"
+#include "SendTritium.h"
+
```

Comment:
Should this file not define the test macro? or is that defined somewhere else?

Reply:
Defined in SendTritium.h. We can define here if that's more beneficial (it makes more sense to me but idk if it will work).